### DISCUSSION TITLE
[Feature] VOB file support

### DISCUSSION THREAD

**Discussion by raminrahimzada** [immich-app/immich#14574] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Today I have started hosting & using & loving immich
Thank you guys for this great tool (y)

I have realized that when I try to upload `*.vob` file, it is requesting  `/api/server/media-types` endpoint and the response does not contain it in video section
Just wanted to ask if there is any plan for [`*.vob`](https://en.wikipedia.org/wiki/VOB) file support
This file format is used inside CD and DVD disks and I have lots of them
Thanks

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by lukasdotcom** [comment]
#14590 implements support for `*.vob` files. Once the next release is out you should be able to upload them.
